docs/DeveloperGuide.md

```diff
@@ -225,185 +236,193 @@ user in the `s-update` command
     * Pros: It can can be extended more easily if there is a greater variety of drinks to store in the future.
     * Cons: There are not many operations to do with `Drink`s. It is only used to represent a constant set of
      drink types.
-     
+
 ## \[Completed\] Finding sales data of some drinks
-  
+
 Finds specific drinks' sales data feature allows the user to get the sales data of a drink quickly. The command is:
-  
+
 * `s-find KEYWORD [MORE_KEYWORDS]` - Views sales data of drinks with the specified keywords.
-  
+
 #### Completed Implementation
-  
-The completed finds sales data of some drinks mechanism is facilitated by `InputContainsKeywordsPredicate`. It implements 
-Predicate<SalesRecordEntry>. 
+
+The completed finds sales data of some drinks mechanism is facilitated by `InputContainsKeywordsPredicate`. It implements
+Predicate<SalesRecordEntry>.
 
 It exposes to `#Model updateFilteredSalesList(Predicate<SalesRecordEntry> predicate)`.
```

Comment:
Maybe should be "It is exposed in the `Model` interface as `Model#updateFilteredSalesList(Predicate<SalesRecordEntry> predicate)` ? I think "is exposed to" and "exposes to" are different?